juntagrico/admins/area_admin.py

```diff
@@ -5,7 +5,7 @@
 class AreaAdmin(BaseAdmin):
     filter_horizontal = ['members']
     raw_id_fields = ['coordinator']
-    list_display = ['name', 'core', 'hidden', 'coordinator']
+    list_display = ['name', 'core', 'hidden', 'coordinator', 'get_email']
```

Comment:
wieso hier die email reinnehmen?

Reply:
kann sich von der coordinator email unterscheiden.

Reply:
das ist schon so aber meine frage ist was ist der usecase, dass man auf dieser seite die email sehen muss?

Reply:
Wenn jemand die Organisation verlässt und man auf einen Blick sehen möchte ob deren/dessen E-Mail-Adresse noch irgendwo rumhängt. Auch bei den aktuell versteckten Tätigkeitsbereichen.

Reply:
ja aber zum überprüfen ob die email noch rumhängt müsste man email und nicht get_emails verwenden. weil wenn er die organisatzion verlässt wird er ja als coordinator ausgetragen und dann will man sehen ob seine email noch hart reingeschrieben ist. generell wäre das die relevante information ob das feld gesetzt ist oder nicht

Reply:
Die harte Email wird von get_email bevorzugt: https://github.com/juntagrico/juntagrico/blob/91e871a8ce913bd14cc24dd1478eae98b69259fe/juntagrico/entity/jobs.py#L39
So ist die richtige information sichtbar und man sieht, was das Mitglied sieht/sehen würde.

Reply:
ok, ich bin der meinung, du übersiehst dann eher den fall wo einer seine persönliche email in den bereich geschrieben hat aber wir können es auch so lassen